gradle/testing/randomization.gradle

```diff
@@ -204,6 +204,10 @@ allprojects {
         if (project.ext.has("useSecurityManager")) {
           useSecurityManager = project.ext.get("useSecurityManager")
         }
+        // Solr does not run with the Security Mangager for Java 24+
```

Comment:
```suggestion
        // There is no longer a Security Manager in Java 24
```

Reply:
This is obsolete with the simpler PR applied before.